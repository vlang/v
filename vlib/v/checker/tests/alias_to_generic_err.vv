module main

type Foo = T

fn main() {
}
