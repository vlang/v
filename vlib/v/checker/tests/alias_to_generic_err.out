vlib/v/checker/tests/alias_to_generic_err.vv:3:12: error: unknown aliased type `T`
    1 | module main
    2 | 
    3 | type Foo = T
      |            ^
    4 | 
    5 | fn main() {
